### DISCUSSION TITLE
What did your validation process consist of for pint?

### DISCUSSION THREAD

**Discussion by ghost** [hgrecco/pint#1857] [Q&A]
We are pretty excited about using pint, however, I was looking for more details about how your code has been validated (or is validated) and I couldn't really find anything other than the extensive test suite in the github repo.    Can you share what process you went through to do validation for pint?  I get that there are tests :) but what was the validation approach and how can I take comfort that the conversion package is solid?  I'm keeping in mind the Mars Climate Orbiter Mishap Investigation Phase I Report.  Thanks!

**Comment by hgrecco** [comment]
We have a very extensive test suite with 91% coverage. In addition, the definition files has been improved and validated over the years. Is there any particular aspect that you are interested in testing or validating?
